src/buildercore/terraform.py

```diff
@@ -808,6 +808,32 @@ def _render_eks_workers_role(context, template):
             'role': "${aws_iam_role.worker.name}",
         })
 
+    if context['eks']['efs']:
+        template.populate_resource('aws_iam_policy', 'kubernetes_efs', block={
+            'name': '%s--AmazonEFSKubernetes' % context['stackname'],
+            'path': '/',
+            'description': 'Allows management of EFS resources',
+            'policy': json.dumps({
+                "Version": "2012-10-17",
+                "Statement": [
+                    {
+                        "Effect": "Allow",
+                        "Action": [
+                            "elasticfilesystem:*",
```

Comment:
we can possibly be more specific here, what `Action`s are required? `Resource` is limited anyway to the AWS account and nothing else uses EFS, so not much to do on that field

Reply:
from https://docs.aws.amazon.com/efs/latest/ug/efs-api-permissions-ref.html, i think these permissions below should be ok

elasticfilesystem:DescribeFileSystems 
elasticfilesystem:DescribeMountTargets
elasticfilesystem:DescribeMountTargetSecurityGroups
elasticfilesystem:DescribeTags

Reply:
yes